src/Resources/config/services_test.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <defaults public="true"/>
```

Comment:
Just commited a bunch of changes - but did not get this working. `server` is unexpected - tried it with `env` under te php-key but still not working. Got no clue, how to get rid of the extra config.

Reply:
Had a look into it again and would suggest to solve it with this code in `createKernel()`:
```php
$kernel->addTestCompilerPass(new class implements CompilerPassInterface {
    public function process(ContainerBuilder $container): void
    {
        // Service would be removed because it's unused.
        $container->findDefinition('shivas_versioning.twig.version')->setPublic(true);
    }
});
```

Otherwise we wouldn't test the actual service configuration, but that one in `services_test.xml`.

Reply:
works perfectly: done